src/x11/x11-platform.c

```diff
@@ -1047,6 +1059,19 @@ X11DisplayInstance *eplX11DisplayInstanceCreate(EplDisplay *pdpy, EGLBoolean fro
         inst->supports_implicit_sync = EGL_TRUE;
     }
 
+    env = getenv("__NV_X11_FORCE_IMPLICIT_SYNC");
```

Comment:
We should probably just use `__NV_DISABLE_EXPLICIT_SYNC` here and below for consistency, since that's what the other platform libraries use.